content/en/blog/_posts/2021-11-18-persistentvolume-reclaim-improvements.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: blog
+title: "Kubernetes 1.23 PersistentVolume reclaim improvements"
+date: 2021-11-18T10:00:00-08:00
+slug: kubernetes-1-23-persistentvolume-reclaim-improvements
+---
+
+**Author:** Deepak Kinni (VMware)
+
+[Persistent Volumes](/docs/concepts/storage/persistent-volumes/) (or PVs for short) are
```

Comment:
This should be PersistentVolume (no space) - see https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects